src/electionguard-ui/ElectionGuard.UI/Resources/Styles/EgStyles.xaml

```diff
@@ -215,7 +215,12 @@
                     </Grid>
                 </VerticalStackLayout>
             </Grid>
-            <ContentPresenter Grid.Row="1" />
+            <ScrollView Grid.Row="1">
+                <VerticalStackLayout>
+                    <ContentPresenter />
+                    <Grid HeightRequest="100" />

```

Comment:
Extra space added after the page content so that it is obvious that the end of the page has been hit.